packages/core/src/track-reference/track-reference.utils.ts

```diff
@@ -62,3 +62,29 @@ export function isTrackReferencePinned(
     return false;
   }
 }
+
+/**
+ * Simple way to create a `TrackReference` from its parts.
+ */
+export function trackReference(
+  participant: Participant,
+  source: Track.Source,
+  publication?: TrackPublication,
+  track?: Track,
```

Comment:
is this needed for types? otherwise we could always infer `track` from the publication